app/components/edit_channel_info/edit_channel_info.js

```diff
@@ -327,6 +329,12 @@ export default class EditChannelInfo extends PureComponent {
                                     defaultMessage='(optional)'
                                 />
                             </View>
+                            <Autocomplete
+                                cursorPosition={header.length}
+                                maxHeight={200}
+                                onChangeText={this.onHeaderChangeText}
+                                value={header}
+                            />
```

Comment:
Please add unit test that `Autocomplete` exists on shallow render of `EditChannelInfo` and functionality check when `onChangeText` is called.

Reply:
@hahmadia Like on our DM, I prepared a gist on how this can possibly tested.  Feel free to add/modify - https://gist.github.com/saturninoabril/594020e65093c723e9e5116c1f577b9b

Reply:
@hahmadia bumping here

Reply:
@saturninoabril @enahum 
Thank you for the help and the sample test case provided. I have included that and it successfully passed! 😄